app/models/book_index_state.rb

```diff
@@ -73,6 +73,10 @@ def self.live
     all.reject{ |doc| doc.deleting? }
   end
 
+  def self.created
+    live.select{ |doc| doc.created? }
```

Comment:
Could be `all.select{...}`; small repeated work to reuse `live` here since the `reject` call in it is not useful here.